srt.go

```diff
@@ -111,7 +113,88 @@ func ReadFromSRT(i io.Reader) (o *Subtitles, err error) {
 			o.Items = append(o.Items, s)
 		} else {
 			// Add text
-			s.Lines = append(s.Lines, Line{Items: []LineItem{{Text: strings.TrimSpace(line)}}})
+			if l := parseTextSrt(line); len(l.Items) > 0 {
```

Comment:
You can replace this with `s.Lines = append(s.Lines, parseTextSrt(strings.TrimSpace(line)).Items...)`

Reply:
Thats no bueno, i think u want this?
`s.Lines = append(s.Lines, parseTextSrt(line))`
I just straight up copied this part from the webvtt equivalent, should i change it there too then?

Reply:
You're right, just replace `line` with `strings.TrimSpace(line)` only here, no need to update the webvtt counterpart

Reply:
Thats done inside the function like in webvtt.